modules/hardware/nvidia-jetson-orin.nix

```diff
@@ -32,13 +32,16 @@
     name = "tegra234-p3701-host-passthrough.dtb";
   };
 
+  hardware.opengl.enable = true;
+  hardware.opengl.driSupport = true;
+
   imports = [
     ../boot/systemd-boot-dtb.nix
   ];
 
   # Passthrough Jetson Orin WiFi card
   boot.kernelParams = [
-    "vfio-pci.ids=10ec:c82f"
+    "vfio-pci.ids=10ec:c82f,10de:1cb1,10de:0fb9"
```

Comment:
I think these might not be even needed, the microvm.nix generates a script (systemd unit) to plug the device into VFIO. I think we originally enabled these because we thought it might make the passthrough easier if the device is bound to VFIO already boot time. The microvm.nix seems to generate the scripts anyway